common/src/main/java/org/openmrs/analytics/OpenmrsUtil.java

```diff
@@ -64,6 +74,9 @@ public Resource fetchFhirResource(String resourceUrl) {
 		}
 	}
 	
+	/**
+	 * Get the restful IGenericClient
+	 */
```

Comment:
This is not adding much on top of the function signature; please drop or extend it.

Reply:
dropped